posydon/binary_evol/DT/magnetic_braking.py

```diff
@@ -0,0 +1,361 @@
+    Rossby_number_pri = Prot_pri / tau_conv_pri
+    Prot_sec = 2 * np.pi / Omega_sec    # [yr]
+    Rossby_number_sec = Prot_sec / tau_conv_sec
+
+    # critical rotation rate in rad/yr
+    Omega_crit_pri = np.sqrt(
+        const.standard_cgrav * M_pri * const.msol
+        / ((R_pri * const.rsol) ** 3)) * const.secyer
+    Omega_crit_sec = np.sqrt(
+        const.standard_cgrav * M_sec * const.msol
+        / ((R_sec * const.rsol) ** 3)) * const.secyer
+
+    # omega/omega_c
+    wdivwc_pri = Omega_pri / Omega_crit_pri
+    wdivwc_sec = Omega_sec / Omega_crit_sec
+
+    gamma_pri = (1 + (wdivwc_pri / 0.072)**2)**0.5
+    T0_pri = K * R_pri**3.1 * M_pri**0.5 * gamma_pri**(-2 * 0.22)
+    gamma_sec = (1 + (wdivwc_sec / 0.072)**2)**0.5
+    T0_sec = K * R_sec**3.1 * M_sec**0.5 * gamma_sec**(-2 * 0.22)
+
+    if (Rossby_number_sec < 0.14):
+        dOmega_mb_sec = (
+            T0_sec * (chi**2.6) * (Omega_sec / omega_sol) / I_sec
+            * np.clip((1.5 - M_sec) / (1.5 - 1.3), 0, 1)
+        )
+    else:
+        dOmega_mb_sec = (
+            T0_sec * ((tau_conv_sec/tau_conv_sol)**2.6)
+                   * ((Omega_sec/omega_sol)**(2.6 + 1)) / I_sec
+            * np.clip((1.5 - M_sec) / (1.5 - 1.3), 0, 1)
+        )
+
+    if (Rossby_number_pri < 0.14):
+        dOmega_mb_pri = (
+            T0_pri * (chi**2.6) * (Omega_pri / 2.6e-6) / I_pri
+            * np.clip((1.5 - M_pri) / (1.5 - 1.3), 0, 1)
+        )
+    else:
+        dOmega_mb_pri = (
+            T0_pri * ((tau_conv_pri/tau_conv_sol)**2.6)
+                   * ((Omega_pri/omega_sol)**(2.6 + 1)) / I_pri
+            * np.clip((1.5 - M_pri) / (1.5 - 1.3), 0, 1)
+        )
+
+    return dOmega_mb_sec, dOmega_mb_pri
+
+
+def calculate_magnetic_braking_G18(p_pri, p_sec):
+    """Use the Garraffo et al. (2018) prescription.
+
+    Calculate the impact of magnetic braking on a binary using the
+    Garraffo et al. 2018, ApJ, 862, 90 prescription. We adopt the following
+    constants:
+    a = 0.03
+    b = 0.5
+
+    Parameters
+    ----------
+    p_pri, p_sec : two tuples
+        The properties of the primary and secondary stars, respectively. Each
+        tuple contains, in order, M_pri (the star's mass in Msun), R_pri
+        (the star's radius in Rsun), Omega_pri (the star's angular velocity in
+        1/yr), I_pri (the star's moment of inertia in Msun Rsun^2),
+        tau_conv_pri (the convective timescale of the primary),
+        and Mdot_pri (the mass loss rate of the star in Msun/yr)
+
+    Returns
+    -------
+    dOmega_mb_sec, dOmega_mb_pri : float
+        The spin angular velocity derivative for the secondary and primary
+        stars, respectively [units of yr^-2]
+
+    """
+    M_pri, R_pri, Omega_pri, I_pri, tau_conv_pri, Mdot_pri = p_pri
+    M_sec, R_sec, Omega_sec, I_sec, tau_conv_sec, Mdot_sec = p_sec
+
+    a_const = 0.03
+    b_const = 0.5
+
+    # [g cm^2] -> [Msol Rsol^2]
+    c = 3e41 / (const.msol * const.rsol**2)
```

Comment:
```suggestion
    c = -3e41 / (const.msol * const.rsol**2)
```

The same as above for M15, the G18 torque currently evaluates to positive. This constant should have a negative to correct that.